md/bonita-layout.md

```diff
@@ -0,0 +1,20 @@
+# Bonita Layout
+
+::: info
+**Note:** For Enterprise, Performance, Efficiency, and Teamwork editions only.
+:::
+
+This layout has different desktop and mobile versions.
+For the dektop version, the layout contains two rows.
+The first row has the logo, the name of the application on the left, as well as the user name, which, when clicked, opens the information about the current session. The last button of the first row opens a window that will let you switch between living applications.
+The second row contains the names of the pages.
```

Comment:
contains the menu of the application pages ?

Reply:
done